src/signup/signup.js

```diff
@@ -89,6 +141,25 @@ class SignupComponent extends React.Component {
       </main>
     );
   }
+
+  formIsValid = () => this.state.password === this.state.passwordConfirmation;
```

Comment:
I'd suggest making the render() method last. Otherwise it's easy to miss these extra methods at the bottom of the file.

Reply:
Updated! Now the render() method is last.